posts/2022-09-01-java-19-22.0.0.10-beta.adoc

```diff
@@ -0,0 +1,341 @@
+</bell>
+----
+
+    Add the keystore for transport security and map a user to the administrator role:
+
+[source, xml]
+----
+<keyStore id="defaultKeyStore" password="keystorePassword" />
+<basicRegistry>
+    <user name="you" password="yourPassword" />
+    <group name="yourGroup" />
+</basicRegistry>
+<administrator-role>
+    <user>you</user>
+    <group>yourGroup</group>
+</administrator-role>
+----
+
+3.Save your changes and start the server
+
+    Look for messages CWWKL0059I and CWWKL0050I in server logs indicating the server enabled BELL SPI visiblity and registered the BELL service. The server makes REST Handler SPI packages visible to a unique classloader for library `RestEpLib`. Applications may declare a common library reference to `RestEpLib` (`commonLibRef="RestEpLib"`) in their `<classloader/>` configuration, but they will not see SPI packages.
+
+[source, xml]
+----
+[...] 00000001 com.ibm.ws.classloading.bells.internal.Bell                  I BETA: BELL SPI Visibility and BELL Properties has been invoked by class com.ibm.ws.classloading.bells.internal.Bell for the first time.
+[...] 00000001 com.ibm.ws.classloading.bells.internal.Bell                  I CWWKL0059I: BELL SPI visibility is enabled for the RestEpLib library. The BELL references the RestEpLib library through a unique class loader that can see SPI packages.
+[...] 00000001 com.ibm.ws.classloading.bells.internal.Bell                  I CWWKL0050I: The RestEpLib library registered the service that is declared in the wsjar:file:/<your_wlp>/usr/servers/bell_ep_server/sharedLib/RestEpLib!/META-INF/services/com.ibm.wsspi.rest.handler.RESTHandler file, using your.org.rest.example.BellEndpoint implementation class.
+----
+
+    When the server registers the BELL service with the OSGi framework it triggers the REST Handler framework to register the service as a listener for URL sub-root `/bellEP`. The /bellEP endpoint is now accessible.
+
+    You can now make calls to `https://<host>:8020/ibm/api/bellEP`.
+
+    The REST handler framework references the BELL service to serve the `/bellEP` endpoint. During the first reference the server creates an instance of class BellEndpoint and injects an unmodifiable map with key/value pair `"hello"/"WORLD"`. The server injects BELL properties at service creation and after BELL configuration updates.
+
+To find out more about this specific update see:
+
+* link:https://openliberty.io/docs/latest/reference/feature/bells-1.0.html[Open Liberty BELL 1.0 feature]
+* link:https://docs.oracle.com/javase/9/docs/api/java/util/ServiceLoader.html[Java ServiceLoader]
+* link:https://openliberty.io/docs/latest/reference/config/bell.html[Open Liberty BELL server configuration]
+
+
+[#security]
+=== Distributed Security Caches
+
+This feature has already been discussed in our Open liberty link:https://openliberty.io/blog/2022/03/17/graphql-22004-beta.html#security_caches[22.0.04-beta] blog where you can find more information reagarding this update. 

```

Comment:
```suggestion
This feature has already been discussed in our Open liberty link:https://openliberty.io/blog/2022/03/17/graphql-22004-beta.html#security_caches[22.0.04-beta] blog where you can find more information regarding this update. 

```

Also there are more details about what should be included in this blog update about this feature here:  https://github.com/OpenLiberty/open-liberty/issues/20209#issuecomment-1217161455 and the follow on comment from @mbroz2.